e2e/filterSkillTest.spec.ts

```diff
@@ -72,23 +70,9 @@ describe('Skill filter', () => {
       }
     }
 
-    await element(by.id('components.mentorList')).swipe('left', 'slow');
-
-    for (let i in mentorIndexes) {
-      try {
-        await expect(
-          element(by.text(accountFixtures.mentors[i].displayName)),
-        ).toBeVisible();
-        mentorsFound[accountFixtures.mentors[i].displayName] = true;
-      } catch (error) {
-        continue;
-      }
-    }
-
```

Comment:
May I ask why this test was changed?

Reply:
Default behaviour changes. Before, it shows all mentors. After, by default it show those not on vacation.

Reply:
I think the requirement was that by default it should show all, and then when enabling this, to hide. It does not feel intuitive by me either, but thats why I remember it

Reply:
Done